### ISSUE TITLE
Request High Priority - Quark Support 1.10.2

### ISSUE THREAD

**Issue by P3rf3ctXZer0** [ShetiPhian/Localizations-and-Issues#28]
Please Read This for Details!
https://github.com/Vazkii/Quark/issues/292

**Comment by ShetiPhian** [comment]
While I'm not sure what your asking for, more on that bellow, I no longer add third party interfaces, InvTweaks is the last one I have and will be removed when possible.
If Vazkii adds a way to do this through FMLInterModComms or Capabilities I'll add support.

-----
I already made changes in MultiStorage to support the Quark buttons, your one screen shot even has them.
To add the support I renamed my Inventory class files so they included chest.

Quark subscribes to the InitGuiEvent, and this line here makes a check to see if the buttons should render
https://github.com/Vazkii/Quark/blob/master/src/main/java/vazkii/quark/management/feature/ChestButtons.java#L62

The check is done here:
https://github.com/Vazkii/Quark/blob/master/src/main/java/vazkii/quark/base/handler/DropoffHandler.java#L96

It first checks if the inventory implements IDropoffManager, which I don't
Then if that fails it checks the name of the class to see if it has chest but not void or trash
My class is named: InventoryStackingChestMulti.class, thus this function will return true, and the buttons should render.

**Comment by P3rf3ctXZer0** [comment]
Oh okay. I am not a coder just a messenger. You two modders should take code with each other not me. I was just relaying that when trying to deposit items to chests with the button in the inventory your mod is not equipped to work. The deposit button completely ignores all "Multi-Storage" chests. Yeah it does work to click the chest first and that is fine if you don't want to code some small changes but just reporting it. I am sure I will not be the last person to say something.

**Comment by ShetiPhian** [comment]
I only implement third party interfaces on classes that are only used for mod support.
Any class that is always in use (Items, Blocks, TileEntities, Inventories, etc.) will not use them.
I know I'm not the only one who has taken this stance, and at some point everyone will.
Capabilities are a much nicer way to add mod support.

-----
I did check through the Quark code again.

Knowing this is a button in the inventory, I located the start and followed code.
A small change in Quark should fix the interaction for any mod that has the buttons on its GUI but the player inventory button fails to work with.

https://github.com/Vazkii/Quark/blob/master/src/main/java/vazkii/quark/base/handler/DropoffHandler.java#L73
This boolean removed, and under the line getting the Capability this line was added:
`boolean accept = isValidChest(player, ret != null ? ret : te);`

-----
There is a possible alternative, refactoring my TileEntities to include Chest in the name should enable the button to work.
But this isn't a game breaking bug so the change will need to wail until the next time I push an update.
